gwtlib/worktrees.py

```
@@ -152,7 +170,113 @@ def switch_branch(branch_name, git_dir, create=False, force_create=False, guess=
+        print(
+            f"Warning: failed to check lock state for {worktree_path}: {e}",
+            file=sys.stderr,
+        )
+        return True  # Fail closed: assume locked if we can't check
+    return False
+
+
+class PreflightError(Exception):
+    """Raised when pre-flight checks fail."""
+
+    pass
```

Ruff (0.15.10)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)